frontend/src/components/membership/index.js

```diff
@@ -0,0 +1,282 @@
+          <ul
+            class="-mb-px flex flex-wrap justify-center text-center text-sm font-medium"
+            id="myTab"
+            data-tabs-toggle="#myTabContent"
+            role="tablist"
+          >
+            <li class="mr-2" role="presentation">
+              <button
+                class="inline-block rounded-t-lg border-b-2 p-4"
+                id="tab-individual"
+                data-tabs-target="#individual"
+                type="button"
+                role="tab"
+                aria-controls="individual"
+                aria-selected="false"
+              >
+                Individual Membership
+              </button>
+            </li>
+            <li class="mr-2" role="presentation">
+              <button
+                class="inline-block rounded-t-lg border-b-2 p-4"
+                id="tab-group"
+                data-tabs-target="#group"
+                type="button"
+                role="tab"
+                aria-controls="group"
+                aria-selected="false"
+              >
+                Group Membership
+              </button>
+            </li>
+          </ul>
+        </div>
+
+        <div id="individual">
+          <h1 class="text-lg font-semibold text-blue-500">
+            Individual Membership
+          </h1>
+          <h3 class="text-sm italic">
+            Renew membership for {player()?.full_name}
+          </h3>
+          <Show
+            when={!membership()?.is_active}
+            fallback={
+              <div id="membership-exist">
+                Membership for {player().full_name} is active until{" "}
+                {displayDate(membership().end_date)}
+              </div>
+            }
+          >
+            {/* <label class="relative inline-flex cursor-pointer items-center">
+            <input
+              type="checkbox"
+              value=""
+              class="peer sr-only"
+              checked={annual()}
+              onChange={() => setAnnual(!annual())}
+            />
+            <div class="peer h-6 w-11 rounded-full bg-gray-200 after:absolute after:left-[2px] after:top-0.5 after:h-5 after:w-5 after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-[''] peer-checked:bg-blue-600 peer-checked:after:translate-x-full peer-checked:after:border-white peer-focus:ring-4 peer-focus:ring-blue-300 dark:border-gray-600 dark:bg-gray-700 dark:peer-focus:ring-blue-800" />
+            <span class="ml-3 text-sm font-medium text-gray-900 dark:text-gray-300">
+              Annual membership
+            </span>
+          </label> */}
+            <Show when={annual()}>
+              <p class="my-4">
+                Pay India Ultimate membership fee (₹ {getAmount()}) valid for
+                the period from {displayDate(season()?.start_date)} to{" "}
+                {displayDate(season()?.end_date)}
+              </p>
```

Comment:
same here, if we can highlight amount and dates in bold / split into new lines by spanning them as `div` would be good i feel

Reply:
![Uploading Screenshot 2024-08-15 at 10.56.19 PM.png…]()